projects/UWMadison_Payseur.yaml

```diff
@@ -0,0 +1,5 @@
+Department: Genetics
+Description: Extending access to OSPool long-term following OSG User School
```

Comment:
Is this the description you want to use? It doesn't say much about what their project is about...

Reply:
@matyasselmeci I can copy something from their lab's website-https://payseur.genetics.wisc.edu/research/ or ask the user Emma to fill in with a better sentence.

Reply:
@ChristinaLK let me know which of the above options that you prefer?

Reply:
@showmic09 I would ask the user for a description, or if there's something that you can take off of the lab's website that you linked

Reply:
Ping @showmic09

Reply:
@brianhlin Totally forgot about that. Pinged the user and if she doesn't reply by Tuesday, I will add a sentence from their lab webpage.

Reply:
@brianhlin The user didn't reply so I made up a sentence based on their research work